execution_time: 1000
image: True
high_cmd: False
control_freq: 25
frame_interval: 1
Kp: 10
Kd: 0.4

